Directory.Build.targets

```diff
@@ -1,6 +1,5 @@
 <Project>
   <PropertyGroup>
-    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(MicrosoftNETCoreApp20PackageVersion)</RuntimeFrameworkVersion>
```

Comment:
Not sure why we didn't update the TFM of a few apps along with the rest of the repo. I tried to remove all targets of netcoreapp2.0. We still need netcoreapp2.1 though.